maintainers/maintainer-list.nix

```diff
@@ -8192,6 +8192,12 @@
     githubId = 1179566;
     name = "Nicolas B. Pierron";
   };
+  pimeys = {
+    email = "julius@nauk.io";
+    github = "pimeys";
+    id = 34967;
+    name = "Julius de Bruijn";
+  };
```

Comment:
This should go in a separate commit titled “maintainers: add pimeys”: https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers

Reply:
My apologizes! Here's the separate commit: https://github.com/NixOS/nixpkgs/pull/129005/commits/190206320cedfdf61698bcf0b47bb3d45c502b60